File: number_of_subarray_of_0s.java
class Solution{
	long no_of_subarrays(int N, int [] arr) {
		//Write your code here
		long ans = 0, count = 0;
        for(int i = 0; i < N; i++){
           if(arr[i] == 0){
               count++;
               ans += count;
           }else{
               count = 0;
           }
        }
        return ans;
	}
}

File: number_of_subarray_of_0s.py
class Solution():
    def no_of_subarrays(self, n,arr):
        #your code goes here
        c=0
        ans=0
        for i in arr :
            if i==0 :
                c +=1
            else :
                ans += c*(c+1)//2 
                c=0
        if c !=0 :
            ans += c*(c+1)//2 
        return ans
        
